install-npm.js

```diff
@@ -37,7 +38,7 @@ async function main() {
     try {
       await exec(npmCommand, ['run', 'build'], {logger: log, cwd: __dirname});
     } catch (e) {
-      throw new Error(`appium-chromedriver package cannot be built: ${e.stderr || e.message}`);
+      throw new Error(`appium-chromedriver package cannot be built: ${util.inspect(e)}`);
```

Comment:
why this change is necessary? why actually want to display stderr there

Reply:
maybe `${('stderr' in e && e.stderr) || ('message' in e && e.message)}` would work

Reply:
Because stderr doesn't have enough details in this case and I just found it shows everything this way.
```
  stdout: '\n' +
    '> appium-chromedriver@5.3.0 build\n' +
    '> tsc -b\n' +
    '\n' +
    "lib/utils.js(134,62): error TS2345: Argument of type '{}' is not assignable to parameter of type 'DriverOpts<Readonly<Record<string, Constraint>>>'.\n" +
    "  Type '{}' is missing the following properties from type 'ServerArgs': address, allowCors, allowInsecure, basePath, and 26 more.\n",
  stderr: 'npm WARN config init.author.name Use `--init-author-name` instead.\n',
```

Reply:
what is an example of the `inspect` output?

Reply:
or maybe then include err.stdout there before stderr?

Reply:
I put the output of inspect() in the PR description.
Including stdout also should be fine